libnetdata/log/log.h

```diff
@@ -44,6 +44,7 @@ extern "C" {
 #define D_ACLK              0x0000000200000000
 #define D_METADATALOG       0x0000000400000000
 #define D_GUIDLOG           0x0000000800000000
+#define D_SQLITE            0x0000001000000000
```

Comment:
not a problem with this PR but just noticed is there any reason why we are skipping `0x0000000080000000` between:
```
#define D_STREAM            0x0000000040000000
#define D_RRDENGINE         0x0000000100000000
```

Reply:
also wouldn't using `D_METADATALOG` serve the purpose? What in case in future netdata uses SQLite in more places will this D_LOG be for all SQLite errors regardles of Netdata part?

Reply:
Hmm good point. Right now the D_SQLITE will report things related to metadata and nothing else (for example generating UUIDs etc)

SQLite errors will be reported in the `error.log` anyway.